src/octopeer-github/content/ContentController.ts

```diff
@@ -105,7 +105,10 @@ class ContentController {
         this.unhookFromDOM();
         this.hookSemanticToDOM(database);
         this.hookTrackersToDOM(database);
-        this.hookMutationObserverToDOM();
+
+        if (Options.get("dataHTML")) {
```

Comment:
This doesn't work yet, right?
The options backend is unchanged yet, which has been done in my PR. It should be `Options.get(Options.DATA_HTML))`.